addon/components/virtual-each/component.js

```diff
@@ -143,6 +136,10 @@ const VirtualEachComponent = Component.extend(EventListenerMixin, DefaultAttrsMi
 
   calculateVisibleItems(positionIndex) {
     run(() => {
```

Comment:
https://github.com/maxmalov/virtual-each/blob/e20d0e23dbd094a594f7d2ed6ae035447b691bb9/addon/components/virtual-each/component.js#L51 is outside the run loop but we can handle that separately I suppose i.e., Ember.run(this, this.calculateVisibleItems)